class MyGoodError(Exception):
    print(" You have an unknown known error")
